opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java

```diff
@@ -94,14 +97,11 @@ public void testAsyncError() {
         List<MockSpan> mockSpans = mockTracer.finishedSpans();
         Assert.assertEquals(1, mockSpans.size());
         assertOnErrors(mockSpans);
-//        TODO resteasy and CXF do not propagate exception to the filter, https://issues.jboss.org/browse/RESTEASY-1758
-//        MockSpan mockSpan = mockSpans.get(0);
```

Comment:
Is this commented out code going to replace the non-commented out once the linked JIRA is fixed? If so, it would probably be better to have two tests, one marked with `@Ignored`.

Reply:
I think it's better, more visible to what needs to be fixed